ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx

```diff
@@ -186,7 +192,7 @@ function PolicyBehaviorForm({ hasActiveViolations }: PolicyBehaviorFormProps) {
                         isRequired
                         touched={touched}
                         helperText={
-                            'Choose lifecycle stage to which your policy is applicable. You can select more than one stage.'
+                            'Choose the lifecycle stage to which your policy is applicable. Image-only policies can be associated with both Build and Deploy stages.'
```

Comment:
[Something weird happened, and my comment did not get submitted with my review, so apologies if you see this comment twice]
Give the updated blue blurb, we can avoid repetition by modifying this helper text to just "Choose the lifecycle stage to which your policy is applicable.".